css/style.css

```diff
@@ -0,0 +1,64 @@
+*{
```

Comment:
I would be careful when you are using * to style because it may mess things that you haven't thought of up.